changelog.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## 0.3.10
+
+Creates new Debian package `securedrop-keyring` for managing the SecureDrop
+Release Signing Key. Rotates the signing key currently in use by setting
+a dependency on the other Debian packages, so that currently running deployments
+will have their apt keyrings updated via automatic nightly updates.
```

Comment:
Since release 0.3.10 rotates the signing key, rotation of the signing key should be included in the changelog. We could omit mention of the new package dependencies, but in light of the expiration that happened in October 2015, where we required Admin intervention on all instances, it makes sense to include mention of the approach that enables us to update currently running instances automatically.